packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -291,7 +287,10 @@ export class ConnectionService {
     // as the recipient key(s) in the connection invitation message
     const signerVerkey = message.connectionSig.signer
 
-    const invitationKey = Key.fromFingerprint(outOfBandRecord.getTags().recipientKeyFingerprints[0]).publicKeyBase58
+    const invitationRecipientKeys = outOfBandRecord
+      .getTags()
+      .recipientKeyFingerprints.map((fingerprint) => Key.fromFingerprint(fingerprint))
+    const invitationKey = invitationRecipientKeys[0]?.publicKeyBase58
```

Comment:
This will transform all keys, but we'll only use the first one. Is there a reason for this change?

Reply:
reverted